src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/model/destination/Chime.kt

```diff
@@ -45,6 +48,14 @@ data class Chime(val url: String?) : ToXContent {
                 .endObject()
     }
 
+    constructor(sin: StreamInput) : this(
```

Comment:
Similar to other place -- not sure why url is nullable when we require it not ot be null in the init, can we just change it to not nullable and not have this optional?